src/Domain/Entities/Physical/Folder.cs

```diff
@@ -4,8 +4,13 @@ namespace Domain.Entities.Physical;
 
 public class Folder : BaseEntity
 {
-    public string Name { get; set; }
-    public string Description { get; set; }
-    public Locker Locker { get; set; }
+    public string Name { get; set; } = null!;
+    public string? Description { get; set; }
+    public Locker Locker { get; set; } = null!;
     public int Capacity { get; set; }
+    public int NumberOfDocuments { get; set; }
+    public bool IsAvailable { get; set; }
+    
+    // Navigation property
+    public ICollection<Document> Documents { get; set; } = new List<Document>();
```

Comment:
not a problem, but is this a feature of EF? first time seeing this, just curious how does it work